package.json

```diff
@@ -104,6 +104,7 @@
     "test-deno": "deno run --allow-env --allow-read --allow-net --allow-run --allow-sys --allow-write ./test/deno.js",
     "test-rs": "START_REPLICA_SET=1 mocha --timeout 30000 --exit ./test/*.test.js",
     "test-tsd": "node ./test/types/check-types-filename && tsd",
+    "test-encryption": "bash scripts/configure-cluster-with-encryption.sh &&  mocha --exit ./test/encryption/*.test.js && npm uninstall mongodb-client-encryption > /dev/null",
```

Comment:
Can we make this command just run `mocha` and move the configure-cluster-with-encryption.sh script to a `setup-encryption-test` script? Sometimes you don't want to re-run configure-cluster-with-encryption.sh when running locally, and also it would be great be able to pass command line flags to Mocha.

Reply:
Sounds good, made the change!